aicsdeformation/bin/deformation_map.py

```diff
@@ -0,0 +1,38 @@
+from argparse import ArgumentParser
+from pathlib import Path
+import pickle
+
+from aicsdeformation.loaders.czi_movie_loader import CziMoveLoader
+from aicsdeformation.aicsdeformation import AICSDeformation
+from aicsdeformation.finishers.cell_deformation_overlays import OverlayFinisher
+
+
+def main():
+    parser = ArgumentParser()
+    parser.add_argument("-ip", "--image_path", help="path to image (czi or tiff)", required=True)
```

Comment:
The normal convention is that only single letter commands have a single `-` in front.

In this case `-ip` should probably be `--ip` to follow conventions.

Reply:
changed to -i